src/strings.rs

```diff
@@ -63,10 +63,10 @@ pub fn normalize_code(v: &[u8]) -> Vec<u8> {
             b'\n' => {
                 r.push(b' ');
             }
-            c => r.push(c),
-        }
-        if v[i] != b' ' {
-            contains_nonspace = true;
+            c => {
+                r.push(c);
+                contains_nonspace = true;
```

Comment:
This obviously is missing an `if c!=b' ' { contains_nonpace = true; }`.
That's why the testis failing. Sorry.

How do I run the spec tests manually?

Reply:
Found the `cibuild` script which contained the commands to run the spec tests.

```
python3 spec_tests.py --program='../../../target/debug/comrak --syntax-highlighting none'
```